programs/fuzz/common.h

```diff
@@ -1,4 +1,9 @@
+#include "mbedtls/build_info.h"
+
+#if defined(MBEDTLS_HAVE_TIME)
```

Comment:
Per my comment on `common.c` above, this should be `#if defined(MBEDTLS_PLATFORM_TIME_ALT)`